services/AuthService.js

```diff
@@ -0,0 +1,38 @@
+const axios = require("axios");
+import { baseUrl, forgotPasswordUrl, resetPasswordUrl } from "../config/config";
+
+async function sendEmail(email) {
+  const item = {
+    email: email
+  };
+  try {
+    const { data } = await axios.post(`${baseUrl}/${forgotPasswordUrl}`, item);
+    // console.log(data);
+    return data;
+  } catch (err) {
+    console.log(err);
+    return;
```

Comment:
if you return nothing from a service, you won't be able to handle the error in the component

Reply:
removed return